src/js/tooltip.js

```diff
@@ -244,6 +244,10 @@ class Tooltip {
 			this.close();
 		}
 
+		if (this.opts.appendToBody) {
```

Comment:
Thanks for the PR, the tooltip element should definitely be removed on `destroy`. I think it should be removed always, not just when `this.opts.appendToBody` is chosen?

Reply:
My instinct is that it should only be removed if the tooltip itself created it, if it was already in the dom it should be left.

Reply:
```suggestion
		if (this.opts && this.opts.content) {
```

Reply:
Maybe the above??